pkgs/applications/science/biology/bayescan/default.nix

```diff
@@ -0,0 +1,36 @@
+{stdenv, fetchurl, unzip}:
+
+stdenv.mkDerivation rec {
+
+  pname = "bayescan";
+  version = "2.1";
+  src = fetchurl {
+    url = "http://cmpg.unibe.ch/software/BayeScan/files/BayeScan${version}.zip";
+    sha256 = "0ismima8j8z0zj9yc267rpf7z90w57b2pbqzjnayhc3ab8mcbfy6";
+  };
+
+  nativeBuildInputs = [ unzip ];
+
+  # Disable FORTIFY_SOURCE or the binary fails with "buffer overflow"
+  hardeningDisable = [ "fortify" ];
+
+  sourceRoot = "BayeScan${version}/source";
+
+  preBuild = ''
+    substituteInPlace Makefile --replace "-static" "";
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mkdir -p $out/share/doc/bayescan
+    cp bayescan_${version} $out/bin
+    cp -r ../*pdf ../input_examples ../"R functions" $out/share/doc/bayescan
+  '';
+
+  meta = {
+    description = "Detecting natural selection from population-based genetic data";
+    license     = "GPL2";
```

Comment:
This should be one of the license attrsets specified in https://github.com/NixOS/nixpkgs/blob/master/lib/licenses.nix - see https://nixos.org/nixpkgs/manual/#sec-meta-license